modules/renter/proto/contractset.go

```diff
@@ -49,6 +49,20 @@ func (cs *ContractSet) Acquire(id types.FileContractID) (*SafeContract, bool) {
 	return safeContract, true
 }
 
+// UpdateContractUtility updates the ContractUtility for a contract with a
+// given id.
+func (cs *ContractSet) UpdateContractUtility(id types.FileContractID, utility modules.ContractUtility) error {
+	cs.mu.Lock()
+	safeContract, ok := cs.contracts[id]
+	cs.mu.Unlock()
+	if !ok {
+		return errors.New("can't update non-existing contract")
+	}
+	safeContract.mu.Lock()
+	defer safeContract.mu.Unlock()
+	return safeContract.updateUtility(utility)
+}
```

Comment:
maybe it makes more sense to make `UpdateUtility` a method on a `SafeContract` that you `Acquire`? Not sure, since the other methods that modify `SafeContract` aren't exported.

Reply:
or perhaps have `cs.UpdateContractUtility` take a `SafeContract` as an argument, like `Delete` and `Renew`.